readme.md

```diff
@@ -187,32 +200,32 @@ The invocation may be immediate or it might happen later. In the example above,
 
 > We can also pass in anonymous functions (i.e., define a nameless function directly inside of the event listener) instead of previously-defined functions.
 
-### You Do: Practice (10 minutes / 0:35)
+### You Do: Practice (2:55 - 3:05, 10 minutes)
 
 Visit this [repository](https://github.com/ga-wdi-exercises/event-listener-practice.git) and follow the instructions.
 
-### Before We Go On... (5 minutes / 0:40)
+### Before We Go On... (3:05 - 3:10, 5 minutes)
 
 Usually when we do anything with functions, we put parentheses after the function name. Here, we have `handleClickEvent` without any parens.
```

Comment:
I think here's a good opportunity to ask/prod the student body about referencing vs invoking and what really is "returned" as the second argument. You don't have to change anything in the lesson. I just think it's a good opportunity for ST-WG